### DISCUSSION TITLE
RG351V: No Network connection with hidden SSID

### DISCUSSION THREAD

**Discussion by MeCias** [351ELEC/351ELEC#689] [Ideas]
Describe the bug
If you try to connect to a hidden WIFI network (hidden SSID) no connection will be established. As soon you make your network visible the connection establishes. 

The behaviour is the same with current release as well as with the beta status 30.09.

Expected behaviour
WIFI with hidden SSID possible

Device (please complete the following information):
 - Version of 351ELEC that you are using "351ELEC - 20210603 Crazy Hedgehog (Hotfix3)" and "beta-20210930_2015"

Thanks a lot guys

**Comment by FridayNightMovies** [comment]
I came to suggest this as a feature update for my RG351P. I wasn't sure if it was a bug or on purpose, but I will go ahead and second this post.
